tests/transports-integrations/tests/integrations/test_google.py

```
@@ -0,0 +1,466 @@
+        genai.configure(api_key=api_key)
+
+    return genai.GenerativeModel(get_model("google", "vision"))
+
+
+@pytest.fixture
+def test_config():
+    """Test configuration"""
+    return Config()
+
+
+def convert_to_google_messages(messages: List[Dict[str, Any]]) -> List[str]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)